docs/setup/settings.asciidoc

```diff
@@ -386,6 +386,11 @@ Specifies an array of trusted hostnames, such as the {kib} host, or a reverse
 proxy sitting in front of it. This determines whether HTTP compression may be used for responses, based on the request `Referer` header.
 This setting may not be used when <<server-compression, `server.compression.enabled`>> is set to `false`. *Default: `none`*
 
+`server.compression.brotli.enabled`::
```

Comment:
Should we add documentation about `quality` as well?

Reply:
Woups, I forgot to add the review comment for that one. Actually no 😅 : I added this setting with the intent of keeping it internal (undocumented) for now, as it was mostly to allow us to eventually perform perf testing on cloud environment tweaking the `quality` value.